lte/gateway/docker/services/c/Dockerfile

```diff
@@ -31,13 +31,17 @@ ENV C_BUILD /build/c
 ENV OAI_BUILD $C_BUILD/oai
 ENV TZ=Europe/Paris
 
-ENV CCACHE_DIR $MAGMA_ROOT/.cache/gateway/ccache
+ENV CCACHE_DIR ${MAGMA_ROOT}/.cache/gateway/ccache
 ENV MAGMA_DEV_MODE 0
-ENV XDG_CACHE_HOME $MAGMA_ROOT/.cache
+ENV XDG_CACHE_HOME ${MAGMA_ROOT}/.cache
 
-# Add the magma apt repo
 RUN apt-get update && \
-    apt-get install -y apt-utils software-properties-common apt-transport-https gnupg
+    # Setup necessary tools for adding the Magma repository
+    apt-get install -y apt-utils software-properties-common apt-transport-https gnupg wget && \
+    # Download Bazel
+    wget -P /usr/sbin --progress=dot:giga https://github.com/bazelbuild/bazelisk/releases/download/v1.10.0/bazelisk-linux-amd64 && \
+    chmod +x /usr/sbin/bazelisk-linux-amd64 && \
+    ln -s /usr/sbin/bazelisk-linux-amd64 /usr/sbin/bazel
```

Comment:
Perhaps it's a very trivial size, but would the bazel binary be required after the base images are built?